src/main/java/build/buildfarm/cas/cfc/CASFileCache.java

```diff
@@ -2008,7 +2008,7 @@ private ListenableFuture<Entry> expireEntry(long blobSizeInBytes, ExecutorServic
       }
       Entry removedEntry = safeStorageRemoval(e.key);
       // reference compare on purpose
```

Comment:
All of these comparisons are intentionally reference compared. Suggest suppression instead, with whatever annotation is necessary.

Reply:
Gosh, I completely missed that comment 🤦